src/rest_framework_jwt/management/commands/obtain_token.py

```diff
@@ -0,0 +1,39 @@
+import sys
+
+from django.core.management.base import BaseCommand, CommandError
+
+
+class Command(BaseCommand):
+    help = 'Create new jwt token for the requested user'
+
+    def add_arguments(self, parser):
+        parser.add_argument('filter', type=str, help='ex: pk=7 or username=apollo or email=foo@bar')
+
+    def _parse_filter(self, filter_value):
+        sep_index = str(filter_value).find('=')
+        if sep_index == -1:
+            raise ValueError("'{}' is not a valid filter".format(filter_value))
+        lookup_field, lookup_value = filter_value[0:sep_index], filter_value[sep_index+1:]
+        return {
+            lookup_field: lookup_value
+        }
+
+    def handle(self, *args, **options):
+        try:
```

Comment:
If this is only for development purposes, why not add an `if settings.DEBUG:` clause that prevents usage in the production environment?

Reply:
Good idea :+1: